include/RED4ext/NativeTypes.hpp

```diff
@@ -56,9 +57,31 @@ struct TweakDBID
     };
 #pragma pack(pop)
 
-    TweakDBID() noexcept = default;
-    TweakDBID(uint64_t aValue) noexcept;
-    TweakDBID(uint32_t aNameHash, uint8_t aNameLength) noexcept;
+    constexpr TweakDBID() noexcept = default;
```

Comment:
Do we care if the value is not inited to 0?

Reply:
Yes :)